src/main/java/Deadline.java

```diff
@@ -0,0 +1,52 @@
+public class Deadline extends Task{
+    protected String by;
```

Comment:
Variable by should be declared as protected since no other classes is extending from Deadline. Consider changing it to a private field instead.